pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -736,14 +736,20 @@ public BundlesData getBundlesData(@PathParam("property") String property, @PathP
             @ApiResponse(code = 412, message = "Namespace is already unloaded or Namespace has bundles activated")})
     public void unloadNamespace(@Suspended final AsyncResponse asyncResponse, @PathParam("property") String property,
             @PathParam("cluster") String cluster, @PathParam("namespace") String namespace) {
-        try {
-            validateNamespaceName(property, cluster, namespace);
-            internalUnloadNamespace(asyncResponse);
-        } catch (WebApplicationException wae) {
-            asyncResponse.resume(wae);
-        } catch (Exception e) {
-            asyncResponse.resume(new RestException(e));
-        }
+        validateNamespaceName(property, cluster, namespace);
```

Comment:
why aren't we wrapping this into a try/catch block as before ?

Reply:
Ah, because `validateNamespaceName ` will throw RestException, it's the same with putting it into try/catch block.

Reply:
@eolivelli So it's no need to add try/catch block.

Reply:
you have asyncResponse, and you should not throw the exception but call
asyncResponse.resume(new RestException(e));

Reply:
yes, I know what you mean.  Before this pr, I have tested locally not added  try/catch block, it’s the same with you comment

Reply:
so throwing RestException is like calling `asyncResponse.resume(new RestException(e));` ?

Reply:
yes, right. However, for better code style and behavior, I still take your opinion.

Reply:
fixed